client/src/pages/Projects.jsx

```diff
@@ -2,16 +2,33 @@
 import React, { useState, useEffect } from 'react';
 import type { Node } from 'react';
 import { useHistory } from 'react-router-dom';
+import { isEqual, findKey, partial } from 'lodash';
+import { Sidebar, Segment, Form, Icon, Modal } from 'semantic-ui-react';
 
 import Page from '../components/layout/Page';
 import Table from '../components/table/Table';
 import { projectColumnDefs } from '../utils/tableHelpers';
-import { getProjects } from '../utils/apiWrapper';
+import {
+  getProjects,
+  getMembers,
+  getUserAuth,
+  updateProject,
+} from '../utils/apiWrapper';
+import { chapterOptions, possibleStatuses } from '../utils/consts';
 
 import '../css/Home.css';
+import '../css/Project.css';
 
 const Projects = (): Node => {
   const [projects, setProjects] = useState([]);
+  const [teamMembers, setTeamMembers] = useState([]);
+  const [visible, setVisible] = useState(false);
+  const [currProj, setCurrProj] = useState({ projectName: '' });
+  const [unmodProj, setUnmodProj] = useState({});
+  const [editable, setEditable] = useState(false);
+  const [editMode, setEditMode] = useState(false);
+  const [showModal, setShowModal] = useState(false);
+  const [teamEmails, setTeamEmails] = useState({});
```

Comment:
nit: these are a lot of states, it might be good to combine some of these into a state object